addons/survey/models/survey_survey.py

```diff
@@ -1028,6 +1029,15 @@ def action_end_session(self):
         self.user_input_ids.sudo().write({'state': 'done'})
         self.env['bus.bus']._sendone(self.access_token, 'end_session', {})
 
+    def action_survey_certified_user_input(self):
```

Comment:
Would defined after action_survey_user_input (just to group them)

Reply:
Replaced by an act_window as you suggested.